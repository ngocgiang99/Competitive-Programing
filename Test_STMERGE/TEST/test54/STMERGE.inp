4
4 3
147537295 441749074 13774365 
861847194 339919525 628469413 
224534301 428010366 829494080 
964703024 286863116 560907085 
4 8
910718959 620596123 93564631 456273666 352466955 222843966 574390877 646590963 
648745445 117219658 301867644 550818927 301505090 909419284 605450265 390440685 
268121640 84255794 450301529 561076256 515776218 986427256 712282784 872513279 
275154399 5117436 672160915 713482429 591102203 746399506 904747033 494963920 
9 3
117156693 583289581 820706252 
168371593 557159084 826913545 
626712652 818903764 914661061 
339674456 732898934 38801030 
117938819 178627665 869497169 
338343219 911819019 220957351 
20121632 886937520 150142403 
304685905 146482282 402260755 
647670352 872687490 172165619 
8 7
255609008 659975361 781669404 530188988 739544208 69783319 624925485 
445758245 868701463 346438011 474427895 922630262 185904250 507145395 
232010448 686462357 678158638 147805635 973460044 695215015 69165031 
421290992 830820161 764506128 664879004 115538111 209113282 80593014 
23716736 233099253 992338471 701697726 189993226 812278167 495212802 
699458828 731139565 598107627 923833130 483056582 651864095 635829620 
792484020 618797968 863700437 323078905 827610099 740452176 175661444 
29140862 77853055 114972715 452437985 698154056 64716842 367772550 
