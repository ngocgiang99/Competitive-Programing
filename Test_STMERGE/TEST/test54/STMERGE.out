13774365
275154399
338769533
29140862
